api/src/db/models/opportunity_models.py

```diff
@@ -385,3 +391,29 @@ class CurrentOpportunitySummary(ApiSchemaTable, TimestampMixin):
         ForeignKey(LkOpportunityStatus.opportunity_status_id),
         index=True,
     )
+
+
+class OpportunityAttachment(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "opportunity_attachment"
+
+    attachment_id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
+
+    opportunity_id: Mapped[int] = mapped_column(
+        BigInteger, ForeignKey(Opportunity.opportunity_id), primary_key=True, index=True
+    )
```

Comment:
Missed this the first time, we don't need opportunity ID to be a part of the primary key here I don't think.

Reply:
Ah, makes sense. Made that change.